bull-transformer-generator/transformer-generator-core/src/test/java/com/hotels/beans/generator/core/mapping/JavaBeanCodeTest.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (C) 2019-2020 Expedia, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.beans.generator.core.mapping;
+
+import static java.lang.String.format;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.containsString;
+
+import org.testng.annotations.Test;
+
+import com.hotels.beans.generator.core.sample.javabean.Destination;
+import com.hotels.beans.generator.core.sample.javabean.Source;
+import com.squareup.javapoet.CodeBlock;
+
+/**
+ * Tests for {@link JavaBeanCode}.
+ * @author mmirk
+ */
+public class JavaBeanCodeTest {
+
+    @Test
+    public void shouldInstantiateAndReturnDestination() {
+        // given
```

Comment:
To keep the code consistency, please put the // given, // when and // then conditions in upper case, and leave an empty row just after all their instructions.